x/wasm/keeper/genesis_test.go

```diff
@@ -133,6 +135,12 @@ func TestGenesisExportImport(t *testing.T) {
 	srcIT := srcCtx.KVStore(wasmKeeper.storeKey).Iterator(nil, nil)
 	dstIT := dstCtx.KVStore(dstKeeper.storeKey).Iterator(nil, nil)
 
+	t.Cleanup(func() {
+		types.MaxWasmSize = 800 * 1024
```

Comment:
Great to do proper housekeeping here! I am a bit afraid that we will easily miss this line whenever the default would be touched in the future. Can you use a temp variable for the original value instead and then use it here? 
like:
```go
origMaxWasmSize := types.MaxWasmSize
types.MaxWasmSize = 1
...
types.MaxWasmSize = origMaxWasmSize
```